selfdrive/car/tesla/readconfig.py

```diff
@@ -31,14 +33,16 @@ def read_config_file(CS):
       file_changed = True
     config.set('OP_CONFIG', 'enable_hso', CS.enableHSO)
 
+    #enable_alca - Enables the Adaptive Lane Change Assist (ALCA) feature which will automatically change lanes when driving above 18 MPH (29 km/h) by just pushing 1/2 way on your turn signal stalk; turn signal will remain on for the duration of lane change
     #enable_alca -> CS.enableALCA
     try:
       CS.enableALCA = configr.getboolean('OP_CONFIG','enable_alca')
     except:
-      CS.enableALCA = True
+      CS.enableALCA = False

```

Comment:
I feel this is better off by default. Thoughts? I think without more sensors we don't want this.